src/Writers/HTMLWriter.jl

```diff
@@ -1030,7 +1030,17 @@ mdconvert(h::Markdown.Header{N}, parent; kwargs...) where {N} = DOM.Tag(Symbol("
 
 mdconvert(::Markdown.HorizontalRule, parent; kwargs...) = Tag(:hr)()
 
-mdconvert(i::Markdown.Image, parent; kwargs...) = Tag(:img)[:src => i.url, :alt => i.alt]
+function mdconvert(i::Markdown.Image, parent; kwargs...)
+    @tags video img a
+
+    if occursin(r"(.webm|.mp4|.ogg|.ogm|.ogv|.avi)$", i.url)
```

Comment:
```suggestion
    if occursin(r"\.(webm|mp4|ogg|ogm|ogv|avi)$", i.url)
```

The unescaped `.` matches any character, so `foomp4` would match currently.

Reply:
Just pushed this from the command line. It seems that you can't accept suggestions even if you have push rights to a PR :man_shrugging: